jena-geosparql/src/main/java/org/apache/jena/geosparql/spatial/SpatialIndex.java

```diff
@@ -457,7 +457,11 @@ public static final Dataset wrapModel(Model model) throws SpatialIndexException
             Resource feature = resIt.nextResource();
 
             Literal lat = feature.getRequiredProperty(SpatialExtension.GEO_LAT_PROP).getLiteral();
-            Literal lon = feature.getRequiredProperty(SpatialExtension.GEO_LON_PROP).getLiteral();
+            Literal lon = feature.getProperty(SpatialExtension.GEO_LON_PROP).getLiteral();
```

Comment:
I notice there's no `null` check for `lat` either

Reply:
The loop is over latitude properties:
```
ResIterator resIt = model.listResourcesWithProperty(SpatialExtension.GEO_LAT_PROP);
```

Reply:
Ah yes, so this is guaranteed to be non-null or we wouldn't be in the loop to start with